src/abstract-authenticated-command.ts

```diff
@@ -0,0 +1,55 @@
+import type Authenticator from './services/authenticator';
+import type * as Config from '@oclif/config';
+
+import AbstractCommand from './abstract-command';
+
+export default abstract class AbstractAuthenticatedCommand extends AbstractCommand {
+  protected readonly authenticator: Authenticator;
```

Comment:
Difficult to read because the method ordering/visibility is not predictable (this, or I don't understand the ordering)